executor/logger/worker.go

```diff
@@ -122,6 +126,26 @@ func (w *Worker) sendKafkaEvent(logReq LogRequest) error {
 }
 
 func (w *Worker) sendCloudEvent(logReq LogRequest) error {
+	var data []byte
+
+	// This temporary fix related to the fact that Triton server responses
+	// are now gzipped compressed. Until we introduce support for gzip
+	// compressed payloads in the logger / adserver and include content-encoding
+	// header in the CloudEvent messages this can serve as temporary solution.
+	if logReq.ContentEncoding == "gzip" {
+		bytesReader := bytes.NewReader(*logReq.Bytes)
```

Comment:
Seems same code as above so better to add to helper function

Reply:
Done!